CodonView.pde

```diff
@@ -5,10 +5,14 @@ class CodonView extends ViewBase {
     final float SEGMENT_SPACING_PERCENTAGE = 0.2;
     final float SEGMENT_CIRCLE_INNER_RADIUS = 10;
     final float SEGMENT_CIRCLE_OUTER_RADIUS = 20;
+    final float SEGMENT_CIRCLE_CENTER_RADIUS = lerp(SEGMENT_CIRCLE_INNER_RADIUS, SEGMENT_CIRCLE_OUTER_RADIUS, 0.5);
+    final float MAX_CODON_WIDTH = 7;
 
     ArrayList<CodonViewClient> clients = new ArrayList<CodonViewClient>();
     CodonBaseModel codonModel;
 
+    float segmentCircleBetweenRadius;
```

Comment:
segmentCircleBetweenRadius can be removed

Reply:
Done